crates/client/db/src/block_db.rs

```diff
@@ -63,37 +62,37 @@ impl DeoxysBackend {
         let col = self.db.get_column(Column::TxHashToBlockN);
         let res = self.db.get_cf(&col, bincode::serialize(tx_hash)?)?;
         let Some(res) = res else { return Ok(None) };
-        let block_n = codec::Decode::decode(&res)?;
+        let block_n = bincode::deserialize(&res)?;
         Ok(Some(block_n))
     }
 
     fn block_hash_to_block_n(&self, block_hash: &Felt) -> Result<Option<u64>> {
         let col = self.db.get_column(Column::BlockHashToBlockN);
         let res = self.db.get_cf(&col, bincode::serialize(block_hash)?)?;
         let Some(res) = res else { return Ok(None) };
-        let block_n = codec::Decode::decode(&res)?;
+        let block_n = bincode::deserialize(&res)?;
```

Comment:
`codec` has been removed, we are now only using bincode, but a previous refactor left this in as it was not a priority.
I removed it here since removing it was faster than converting it to not use StarkFelts.